crawl-ref/source/dat/strings/de/hiscores.txt

```diff
@@ -278,21 +278,21 @@ Vernichtet
 %%%%
 # killed by missile from a launcher (arrow, bolt, sling bullet)
 shot by themself
-ershießen von sich selbst
+erschossen von sich selbst
 %%%%
 shot by %s
-ershießen von {dat}%s
+erschossen von {dat}%s
 %%%%
 # killed by thrown weapon (boomerang, javelin, manticore spikes)
 hit from afar by %s
 aus der Ferne von {dat}%s getroffen
 %%%%
 # wand, etc.
 blasted by themself
-eingeschlagen von sich selbst
+gesprengt von sich selbst
```

Comment:
This may not be an explosion. It could be any type of "beam".

Reply:
Einschlagen doesn't work here. It's what a projectile does, not the one firing it. It also implies a solid projectile, not a ray of energy of some kind. Gesprengt is the best I can think of for an unknown beam. Means whatever you hit yourself with, it was powerful enough to blow you apart.